src/cockpit/channel.py

```diff
@@ -30,12 +31,12 @@ def do_channel_data(self, channel, data):
 
 
 class Channel(Endpoint):
-    payload = None
-    restrictions = ()
+    payload = ""
+    restrictions: Dict[str, str] = {}
```

Comment:
The problem with using a dict here is that a dict is mutable.  It's considered bad form to use mutable values for defaults because if you modify it from one instance, it may not be obvious that you're effectively modifying all instances...

Reply:
Ah, but mypy is not happy about switching types so hmm a solution should be found or we make it `Optional[Dict[str, str]] = None`

Or of course `Set | Dict[str, str] = ()`

Reply:
You can set only the type in the class:
```python
payload: str
restrictions: dict[str, str]
```